src/app/lib.ts

```diff
@@ -26,8 +26,12 @@ export const mapBiggReactionToCobra = (reaction): types.Cobra.Reaction => ({
   name: reaction.name,
   id: reaction.bigg_id || reaction.id,
   gene_reaction_rule: reaction.gene_reaction_rule || '',
-  lower_bound: reaction.lower_bound || defaultBounds.lowerBound,
-  upper_bound: reaction.upper_bound || defaultBounds.upperBound,
+  lower_bound: reaction.hasOwnProperty('lower_bound')
+                ? reaction.lower_bound
+                : defaultBounds.lowerBound,
+  upper_bound: reaction.hasOwnProperty('upper_bound')
+                ? reaction.upper_bound
+                : defaultBounds.upperBound,
```

Comment:
Did you have any errors with this part?

Reply:
Yes, there was a problem, because when for example reaction.lower_bound is 0, it's a falsy value and lower_bound was set to defaultBounds.lowerBound (-1000 instead of 0)